### PR TITLE
Spelling 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by nschonni**
Think this is the last of it. Just took a quick second pass to see what I had accidentally hit "Ignore All" in #2085

### CODE REVIEW

File: src/Shared/StringBuilderCache.cs [hunk 1]
```diff
@@ -5,28 +5,28 @@
 **
 **
 ** Purpose: provide a cached reusable instance of StringBuilder
-**          per thread  it's an optimisation that reduces the 
+**          per thread  it's an optimization that reduces the
```
Comment (rainersigwald) [inline src/Shared/StringBuilderCache.cs]: Who on the MSBuild team was ever British? Huh. :us:

File: src/Tasks.UnitTests/CreateProperty_Tests.cs [hunk 2]
```diff
@@ -47,7 +47,7 @@ public void CreateBlankProperty()
 
         /// <summary>
         /// Make sure that I can use the CreateProperty task to create a property
-        /// that has a parseable semicolon in it.
+        /// that has a parsable semicolon in it.
```
Comment (rainersigwald) [inline src/Tasks.UnitTests/CreateProperty_Tests.cs]: Was skeptical of this change but it does look like a sorta-standard: https://en.wiktionary.org/wiki/parsable.

Comment (nschonni) [inline src/Tasks.UnitTests/CreateProperty_Tests.cs]: Yeah, it was spelled both ways in the repo so I did the same think and did a search to find which was "correct"

**Comment by rainersigwald** [comment]
Thanks again!

File: src/Build.OM.UnitTests/Definition/Project_Tests.cs [hunk 3]
```diff
@@ -81,7 +81,7 @@ public void Dispose()
             ";
 
         /// <summary>
-        /// Since when the project file is saved it may be intented we want to make sure the indent characters do not affect the evaluation against empty.
+        /// Since when the project file is saved it may be intended we want to make sure the indent characters do not affect the evaluation against empty.
```
Comment (eerhardt) [inline src/Build.OM.UnitTests/Definition/Project_Tests.cs]: `indented`?

Comment (nschonni) [inline src/Build.OM.UnitTests/Definition/Project_Tests.cs]: Yup, that would make more sense.

**Pull request merged**